src/Console/Commands/CheckCommand.php

```diff
@@ -198,6 +198,8 @@ private function renderLineIssue(Issue $issue): void
             $issue,
         );
 
+        $suggestionsHelperText = ($suggestions > '') ? 'Did you mean:' : 'No suggestions found.';
```

Comment:
I would prefer checking directly for "not equals" or am I missing something?

```suggestion
        $suggestionsHelperText = ($suggestions !=== '') ? 'Did you mean:' : 'No suggestions found.';
```